Sources/Commands/SwiftBuildTool.swift

```diff
@@ -37,11 +37,11 @@ private enum Mode: Argument, Equatable, CustomStringConvertible {
 
     init?(argument: String, pop: () -> String?) throws {
         switch argument {
-        case "--configuration", "--conf", "-c":
+        case "--configuration", "--config":
```

Comment:
I'm not sure if we should drop `-c` just yet, I think people and scripts are likely to rely on it.